MIGRATION-FROM-SCK.md

```diff
@@ -0,0 +1,325 @@
+# Migration from Splunk Connect for Kubernetes
+
+## Assumptions
+
+You are running Splunk Connect for Kubernetes (SCK) 1.4.9 and want to migrate to Splunk OpenTelemetry Collector for Kubernetes. Previous versions of the SCK are out of scope, but this guide still applies. You might encounter some version specific issues (such as missing/changed config options) but that shouldn't prevent you from proceeding with migration.
+
+## Changes in Components
+
+Splunk Connect for Kubernetes(SCK) has 3 components/applications:
```

Comment:
If SCK is an approved acronym, it should be used in the rest of the document. For example, "SCK has three components and applications:"